specification/httpxe/src/main/scripts/org/kaazing/specification/httpxe/origin/request.with.kac.parameter/request.rpt

```diff
@@ -0,0 +1,14 @@
+connect http://localhost:8000/path
+connected
+
+write parameter ".kac" "ex"
+write method "GET"
+write version "HTTP/1.1"
+write header "Referer" "http://source.example.com:80"

```

Comment:
do we need Referer header? 
The spec says that client can request Access-Control-* header in response by adding .kac query parameter even for same origin requests.
So, the Origin header here should be ``http://localhost:8000``